app/src/androidTest/java/com/popularMovies/tests/BookmarkTests.java

```diff
@@ -14,11 +17,13 @@
  */
 
 public class BookmarkTests extends EspressoTestBase {
+
     @Test
     public void testiIfBookmarkedMoviesIsEmpty() throws Exception {
         NavDrawer.bookmarkedMovies();
-        Helpers.isItemDisplayed("NO BOOKMARKS ADDED");
+        Assert.assertTrue(Strings.NO_BOOKMARKS, BookmarkMovie.isNoBookmarksAdded());
```

Comment:
Sigur trece testul? :-?

Ce reprezinta "Strings.NO_BOOKMARKS"? Acel parametru trebuie sa fie mesajul de eroare care o sa fie returnat in caz ca pica assert-ul. ex: "The title should be displayed."